appdata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The is already an AppStream metadata file in the upstream source tree:

tarball: `pidgin-2.13.0/pidgin/data/pidgin.appdata.xml.in`

repository:
https://bitbucket.org/pidgin/main/src/0b7a0190d83db173f80755c2d6267b0811cc07eb/pidgin/data/im.pidgin.Pidgin.appdata.xml.in